tests/typesets/test_standard_set_sparse.py

```diff
@@ -38,12 +38,23 @@
         "bool_sparse",
         "complex_sparse",
         "str_obj_sparse",
+        "pd_bool_sparse",
+        "pd_string_sparse",
     },
 }
 
-if int(pd.__version__.split(".")[0]) >= 1:
-    contains_map[Generic].add("pd_bool_sparse")
-    contains_map[Generic].add("pd_string_sparse")
+
+def remove_key(map, key):
```

Comment:
Are you changing this to fix the mypy error?

Simpler would be to the variable annotations (PEP 526), introduced in py3.6:
`contains_map: Dict[Type[VisionsBaseType], Set[str]]`
(which we couldn't use before when supporting 3.5)